### PR TITLE
capstone project to review 

### PR DESCRIPTION
**Pull request by frnega**
hello 
this pull request included all the requirements from the capstone project,

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi Team, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't show reviews given to each other as per the project requirements. Kindly make sure both give reviews to each other.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by frnega — COMMENTED** [review]
Hi @ifeanyi305 ,

Good job so far! 🎆
Alt Text
There are no issues you need to work on well done.

Highlights
Great working on the project required 💡
used Gitflow 
webpack configured correctly
All linters passed 👍🏼
Styling ✅
Readme file ✔️


Required Changes ♻️
nothing to change 

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

File: src/modules/comment.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      fetch(`${baseURL}${button.id}`)
+        .then((response) => response.json())
+        .then((elem) => {
+          CommentModal.classList.remove('hidden');
+          modalImage.setAttribute('src', elem.meals[0].strMealThumb);
+          mealName.innerHTML = elem.meals[0].strMeal;
+          recipe.innerHTML = elem.meals[0].strInstructions;
+        });
+    });
+  });
```
Comment (Abenezer-Tilahun) [inline src/modules/comment.js:51]: - [ ] As stated in the project requirement for comments Popup, When the user clicks on the "Comment" button, the data is recorded in the Involvement API and **the screen should be updated before I reload page**  but to get the submitted comments I have to reload the page as shown in below recorded video. So Kindly Fix it.

- Check out The video [Link](https://drive.google.com/file/d/1YLv80bd-rmxIqWQlhJQ6wxVVv8mDlxMh/view?usp=share_link)

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @frnega  @ifeanyi305    , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional  README 
✔️ Good commit messages
✔️ All Test are Passing

almost

## Required Changes ♻️

- [ ]  As stated the project requirement you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. it's one of the project requirement you must fulfill for to get approval for project.

Capture

- [ ] Good Job using new [README template](https://github.com/microverseinc/readme-template) and customize according to your project But Kindly I recommend you to added the section "Future Features", including **1-3 potential features to be added**.  


_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi team, 

In my opinion, your project is complete! 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good design
- Working functionality.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]

**Pull request merged**